SurveyTests/Sources/Mocks/UserSession/UserSessionUseCaseMock.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UserSessionUseCaseMock.swift
+//  SurveyTests
+//
+//  Created by Khanh on 16/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+@testable import Survey
+
+import Combine
+
+final class UserSessionUseCaseMock: UserSessionUseCaseProtocol {
+
+    var hasUserLoggedInCalled = false
+
+    var hasUserLoggedInReturnValue = Observable.just(true)
```

Comment:
Please add a comment to indicate that this will return true by default or use force unwrap to reduce ambiguity.
```
/// Default is `true`
var hasUserLoggedInReturnValue = Observable.just(true)
```
or
```
var hasUserLoggedInReturnValue: Observable<Bool>!
```